### PR TITLE
Add wasm target.

### PR DESCRIPTION
**Pull request by terrakok**
image

### CODE REVIEW

File: src/jvmMain/kotlin/Main.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    val state = rememberWindowState(position = WindowPosition.Aligned(Alignment.Center))
+    Window(
+        title = "Sandbox",
+        onCloseRequest = ::exitApplication,
+        resizable = false,
+        state = state,
+    ) {
+        app()
+    }
+}
```
Comment (rock3r) [inline src/jvmMain/kotlin/Main.kt]: Nit: missing empty line at end of file — pretty much in all new files :)

**Comment by rock3r** [comment]
Thanks @terrakok this is awesome!

File: src/jvmMain/kotlin/Main.kt (hunk 1 above)
Comment (terrakok) [reply src/jvmMain/kotlin/Main.kt]: done

**Review by rock3r — APPROVED** [review]
Thanks! <3

**Pull request merged**